values/gitea/gitea.gotmpl

```diff
@@ -4,7 +4,21 @@
 {{- $k := $v.apps.keycloak }}
 {{- $giteaDomain := printf "gitea.%s" $v.cluster.domainSuffix }}
 {{- $cm := $v.apps | get "cert-manager" }}
+{{- $ENV_DIR := env "ENV_DIR" | default "../env" }}
+{{- $t := readFile (print $ENV_DIR "/env/teams.yaml") | fromYaml }}
+{{- $teamData := dict }}
+{{- $teamConfigKeys := keys $t.teamConfig }}
+{{- if gt (len $teamConfigKeys) 0 }}
+  {{- range $team := $teamConfigKeys }}
+  {{- if ne $team "admin" }}
+    {{- $teamPath := printf "/team-%s" $team }}
+    {{- $teamName := printf "team-%s" $team }}
 
+    {{- $teamData = merge $teamData (dict $teamPath (dict "otomi" (list "otomi-viewer" $teamName))) }}
+  {{- end }}
+  {{- end }}
+  {{- $teamData = toJson $teamData }}
```

Comment:
It does not seem to be necessary here. I would rather use the `toJson` function at line 128